### PR TITLE
New /cgi/css.pl to serve LTR or RTL CSS based on subdomain language

### PR DESCRIPTION
**Pull request by stephanegigandet**
fixes #6082

An alternative would be to try to hardcode all subdomains + languages that need RTL, and have some JS code to select the CSS.

But this should be good enough.

### CODE REVIEW

**Comment by teolemon** [comment]
It would be good to expand on this for langcodes/countries/language dependant assets. That would massively simplify translations drastically, and massively improve reliability as well.
https://world.openfoodfacts.org/images/svg/openfoodfacts-logo-en.svg
https://world.openfoodfacts.org/images/misc/appstore/black/appstore_US.svg
https://static.openfoodfacts.org/images/misc/google-play-badge-svg-master/img/en_get.svg
https://world.openfoodfacts.org/images/misc/microsoft/English.svg
https://world.openfoodfacts.org/images/svg/fairtrade-origins-min.svg

**Comment by stephanegigandet** [comment]
> It would be good to expand on this for langcodes/countries/language dependant assets. That would massively simplify translations drastically, and massively improve reliability as well.

This PR is for static pages. For dynamic pages, making a new cgi request for each file we wan't to localize is not a very good option.

This is a quick fix for the static donation pages, so that we can merge CrowdIn PR requests as-is.

**Review by alexgarel — APPROVED** [review]
LGTM - now you have a lot more html pages to change, isn'it ?

File: cgi/css.pl [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+use Apache2::Const -compile => qw(OK);
+use CGI qw/:cgi :form escapeHTML/;
+use URI::Escape::XS;
+use Encode;
+use Log::Any qw($log);
+
+ProductOpener::Display::init();
+
+# Redirect the the left to right or right to left CSS based on the subdomain
```
[the remark this answers is not in the document]
Comment (stephanegigandet) [reply cgi/css.pl:41]: > LGTM - now you have a lot more html pages to change, isn'it ?

Crowdin should regenerate them from the en.html file.

**Pull request merged**